docs/guide/features.md

```diff
@@ -292,7 +292,23 @@ import MyWorker from './worker?worker'
 const worker = new MyWorker()
 ```
 
-The worker script can also use `import` statements instead of `importScripts()` - note during dev this relies on browser native support and currently only works in Chrome, but for the production build it is compiled away.
+Worker options are supported using url query. avalable options are
+```ts
+type RequestCredentials  = "omit" | "same-origin" | "include"
+type WorkerType  = "classic" | "module"
+
+interface WorkerOptions {
+  credentials?: RequestCredentials;
+  name?: string;
+  type?: WorkerType;
+}
+```
+example url query
+```js
```

Comment:
**nitpick (non-blocking, formatting):** Seems that prettier didn't formatted this well
Could you add blank lines between the code blocks and the line between them?
Also could you use single quotes for the literal type values?
Maybe you can just run prettier locally and it may fix all or some of these :slightly_smiling_face: